README.md

```diff
@@ -42,6 +42,7 @@ dom.switchClass(el, 'oldclass', 'newclass');
 // makes sure attribute (with no content) is added
 // if exists it will be cleared of content
 dom.addAttribute(el, 'checked');
+dom.hasAttribute(el, 'checked'); // => true
```

Comment:
Maybe add a comment before this explaining what it is and that it returns a boolean?